python/ray/rllib/models/visionnet.py

```diff
@@ -17,10 +17,14 @@ def _init(self, inputs, num_outputs, options):
             [32, [4, 4], 2],
             [512, [10, 10], 1],
         ])
+        conv_activation = options.get("conv_activation", "relu")
+        activation = tf.nn.__dict__[conv_activation]
```

Comment:
oh nice; is this standard?

Reply:
it python :)
I found this solution quite powerful and easy to go

Reply:
by this way, it's possible to use any tf activation functions tanh/elu/whatever without any performance troubles